### DISCUSSION TITLE
Option to have on-screen keyboard appear automatically

### DISCUSSION THREAD

**Discussion by pbienst** [gsantner/markor#2042] [General] [answered]
I just upgraded from 2.8.1, and in that version, as soon as you opened a file in edit mode, the on-screen keyboard would appear. The current version no longer does that, which slows down my work flow. Is there any possibility this could be brought back, perhaps as a configurable option?

**Comment by harshad1** [comment, marked as answer]
IMO this should be the behavior for todo and quicknote alone. Making it configurable is unnecessary.
